docs/gmi.md

```diff
@@ -203,6 +203,10 @@ gmi.account.signIn
 
 This will perform a full-page re-direct to a sign-in form. Once the user has completed sign-in they will be returned to the page from which they left to complete the sign-in form.
 
+#### Sign-in on Pick'n'Mix
+
+When in Pick'n'mix, `account.signIn` will open a new window with the BBC ID sign in, rather than redirect the user. When the user has successfully signed in, this will return with a promise. This promise should be handled correctly by the client inside PnM, but can be ignored when on web.
```

Comment:
Can we make sure it's clear this includes register as well?

Reply:
done